duckduckgo/Android
# Change app icon

Task/Issue URL: https://app.asana.com/0/72649045549333/1161651092205713

**Description**:
The iOS team saw a positive response from users after adding the feature that enabled users to change the app icon. We think that the same outcome can be achieved on the Android side.

**Disclaimer**:
It's known that this feature won't work on all devices. Depending on the launcher and OEM, it's possible that after selecting an icon it closes the app. If it doesn't, be aware that the icon change may take a few seconds before it's applied.

**Steps to test this PR**:
1. Open app 
2. Open Settings and Change App Icon
3. Select a new icon
4. Navigate to Apps Menu
5. Check new icon has been applied

**Screenshots**:
| Settings screen | Change Icon screen |
| ------ | ----- |
![device-2020-02-28-164625](https://user-images.githubusercontent.com/531613/75897758-5d51a500-5e39-11ea-8378-e93a200ee433.png)|![device-2020-03-04-103920 (1)](https://user-images.githubusercontent.com/531613/75897762-5e82d200-5e39-11ea-98d3-6d57b0d26ff1.png)
